data/data/bootstrap/baremetal/files/etc/containers/systemd/ironic.container.template

```diff
@@ -18,6 +18,10 @@ Exec=/bin/runironic
 Network=host
 Volume=${AUTH_DIR}:/auth:z,ro
 Volume=ironic.volume:/shared:z
+{{ if eq .PlatformData.BareMetal.DisableIronicVirtualMediaTLS false }}
+Volume=/etc/ironic/ironic-virtualmedia-tls.crt:/certs/vmedia/tls.crt:z
+Volume=/etc/ironic/ironic-virtualmedia-tls.key:/certs/vmedia/tls.key:z
```

Comment:
nit: with some strategic filename changes you could probably get this down to a single directory mount.

Reply:
Done.